example/app/models.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+import saritasa_s3_tools.django
+
+
+class User(AbstractUser):
+    """User model."""
+
+
+class ModelWithFiles(models.Model):
+    """Test model with different files configs."""
+
+    file = saritasa_s3_tools.django.models.S3FileField(
+        blank=True,
+        null=True,
+        s3_config=saritasa_s3_tools.S3FileTypeConfig(  # type: ignore
```

Comment:
Can you improve typing to avoid `# type: ignore`?

Reply:
![image](https://github.com/user-attachments/assets/0183991a-1740-4b97-9fc3-4a2a5197c39e) It's pylance thing, mypy is okay. No idea how to make it okay for it

Reply:
Maybe use `# pyright: ingore` then?

Reply:
Ok that works, weird that vscode doesn't suggest that